styles.css

```diff
@@ -111,3 +108,452 @@ li {
   margin: 15px;
   margin-left: 0;
 }
+
+/* Grid container */
+
+.grid-position {
+  display: flex;
+  justify-content: center;
+  margin-top: 180px;
+}
+
+.Grid-container {
+  display: grid;
+  width: 375px;
+  grid-template-columns: auto;
+  background: #e5e5e5;
```

Comment:
- [ ] [OPTIONAL] It would be better you change the background color of this section as it does not align with the original design.

![image](https://user-images.githubusercontent.com/43995830/154797555-48b59403-15ef-490f-975e-b7c0603c5d3f.png)